sympy/assumptions/tests/test_query.py

```diff
@@ -2402,10 +2402,11 @@ class MyPredicate(Predicate):
 
 
 def test_relational():
-    assert ask(Q.eq(x, 0), Q.zero(x))
-    assert not ask(Q.eq(x, 0), Q.nonzero(x))
-    assert not ask(Q.ne(x, 0), Q.zero(x))
-    assert ask(Q.ne(x, 0), Q.nonzero(x))
```

Comment:
I changed these because `not ask(Q.ne(x, 0), Q.zero(x))` will pass the assertion if the result is None or False.

This didn't seem intentional, but it might have been.

Reply:
It is generally better to check `is True` as you have done here.

Aside from `not` accepting both `None` and `False` asserting something like `assert f()` is not very specific because almost any kind of object returned by `f` would pass the test so `assert f() is True` is better.